erpnext_shopify/sync_products.py

```diff
@@ -423,7 +423,7 @@ def update_item_stock_qty():
 		update_item_stock(item.item_code, shopify_settings)
 
 def update_item_stock(item_code, shopify_settings, bin=None):
-	item = frappe.get_doc("Item", item_code)
+	item = frappe.get_doc("Item", {"item_code": item_code})
```

Comment:
Why should the name be shopify id and not item code?

Best,
Anand Doshi

ERPNext

> On 27-Jan-2016, at 4:29 PM, Saurabh <notifications@github.com> wrote:
> 
> In erpnext_shopify/sync_products.py <https://github.com/frappe/erpnext_shopify/pull/46#discussion_r50968954>:
> 
> > @@ -423,7 +423,7 @@ def update_item_stock_qty():
> >  		update_item_stock(item.item_code, shopify_settings)
> >  
> >  def update_item_stock(item_code, shopify_settings, bin=None):
> > -	item = frappe.get_doc("Item", item_code)
> > +	item = frappe.get_doc("Item", {"item_code": item_code})
> @anandpdoshi <https://github.com/anandpdoshi>
> While syncing from shopify we keep Item Code and Name name as shopify id.
> 
> but while pushing data from ERPNext to Shopify, I am using filter here in case if item code is not name.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/frappe/erpnext_shopify/pull/46/files#r50968954>.
>